lib/components/recordingSession/recorder.dart

```diff
@@ -30,9 +31,9 @@ class _RecorderState extends State<Recorder> {
   MyWebSocket ws;
   Directory directory;
   bool _isRecording = false;
-  StreamSubscription _recorderSubscription;
-  StreamSubscription _dbPeakSubscription;
-  StreamSubscription _playerSubscription;
+  StreamSubscription<RecordStatus> _recorderSubscription;
+  StreamSubscription<double> _dbPeakSubscription;
```

Comment:
_dbPeakSub.. isn't that readable. Do you mean changing the variable so that we can have a better time understanding your code?

Reply:
I will look into this